docs/04-start-server.md

```diff
@@ -0,0 +1,70 @@
+---
+id: cli
+title: Using the CLI
+layout: docs
+permalink: /docs/cli.html
+prev: cli.html
+---
+
+The flow command line tool is made to be easy-to-use for the simple case.
+
+Just using the command `flow` will type-check your current directory if the `.flowconfig` file is present.
```

Comment:
Woohoo! It seems like many people assume `flow check` to be the default when really `flow` should be the common case. Hopefully these docs will help clarify this for everyone.